src/test/scala/scala/concurrent/stm/examples/SyntaxCheatSheet.scala

```diff
@@ -5,9 +5,9 @@ package scala.concurrent.stm.examples
 object SyntaxCheatSheet {
   import scala.concurrent.stm._
 
-  val x = Ref(0) // allocate a Ref[Int]
```

Comment:
Depending on the use case of the cheat sheet, perhaps we should not have explicit types here?